src/core/typing.ts

```diff
@@ -176,15 +164,13 @@ export class DefaultTyping
    * @inheritDoc
    */
   get(): Promise<Set<string>> {
-    return this._channel.then((channel) =>
-      channel.presence.get().then((members) => new Set<string>(members.map((m) => m.clientId))),
-    );
+    return this._channel.presence.get().then((members) => new Set<string>(members.map((m) => m.clientId)));
```

Comment:
Could we change this to an async function, seems to be our prefered style. See start/stop below.

Also, not related, but do you think it'd be useful to have a trace log here?

Reply:
We only tend to use `async` when there's something to `await`, so the style here is consistent